src/getting-started/setup.md

```diff
@@ -45,11 +45,11 @@ export PATH=/Library/Developer/Toolchains/swift-latest.xctoolchain/usr/bin:"${PA
 
 ```bash
 $ swift --version
-SwiftWasm Swift version 5.6 (swiftlang-5.6.0)
-Target: x86_64-apple-darwin20.6.0
+SwiftWasm Swift version 5.7.1 (swiftlang-5.7.1)
+Target: x86_64-apple-darwin21.6.0
 ```
 
-> Warning: `xcrun` finds executable binary based on `--toolchain` option or `TOOLCHAINS` environment variable, but it also sets `SDKROOT` as host target SDK (e.g. `MacOSX.sdk`). So you need to specify `-sdk` option as `/Library/Developer/Toolchains/swift-wasm-5.6.0-RELEASE.xctoolchain/usr/share/wasi-sysroot` when launching `swiftc` from xcrun. `swift build` or other SwiftPM commands automatically find SDK path based on target triple, so they don't require to specify it.
+> Warning: `xcrun` finds executable binary based on `--toolchain` option or `TOOLCHAINS` environment variable, but it also sets `SDKROOT` as host target SDK (e.g. `MacOSX.sdk`). So you need to specify `-sdk` option as `/Library/Developer/Toolchains/swift-wasm-5.7.1-RELEASE.xctoolchain/usr/share/wasi-sysroot` when launching `swiftc` from xcrun. `swift build` or other SwiftPM commands automatically find SDK path based on target triple, so they don't require to specify it.
 
 
 ## Linux
```

Comment:
I have checked apt dependencies and there are no changes between 5.6 and 5.7

https://github.com/swiftwasm/swiftwasm-docker/blob/main/5.7/ubuntu/20.04/Dockerfile
https://github.com/swiftwasm/swiftwasm-docker/blob/main/5.6/ubuntu/20.04/Dockerfile
https://github.com/swiftwasm/swiftwasm-docker/blob/main/5.7/ubuntu/18.04/Dockerfile
https://github.com/swiftwasm/swiftwasm-docker/blob/main/5.6/ubuntu/18.04/Dockerfile